releases/release-1.31/release-notes/maps/pr-122176-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 122176
+releasenote:
+  text: 'kube-apiserver: http/2 serving can be disabled with a `--disable-http2-serving`
+    flag'
```

```suggestion
  text: 'kube-apiserver: Added support to disable http/2 serving with a `--disable-http2-serving` flag.
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653154562